rosdep/python.yaml

```diff
@@ -1325,6 +1325,13 @@ python-debian:
     yakkety_python3: [python3-debian]
     zesty: [python-debian]
     zesty_python3: [python3-debian]
+python-decorator:
+  debian: [python-decorator]
+  fedora: [python2-decorator]
```

Comment:
The Python 2 subpackage isn't available for any non-EOL Fedora releases. Let's just drop it.

```suggestion
```

Reply:
Done as requested